docs/pages/rows/row_dragging.py

```diff
@@ -12,42 +12,319 @@
-- Unmanaged Dragging: This is more complex and more powerful. The grid will not rearrange rows as you drag. Instead the application is responsible for responding to events fired by the grid and rows are rearranged by the application.
+- Unmanaged Dragging: This is more complex and more powerful. The grid will not rearrange rows as you drag. Instead the
+  application is responsible for responding to events fired by the grid and rows are rearranged by the application.
 
 ## Managed Dragging
-In managed dragging, the grid is responsible for rearranging the rows as the rows are dragged. Managed dragging is enabled with the property rowDragManaged=true.
+
+In managed dragging, the grid is responsible for rearranging the rows as the rows are dragged. To enable the managed
+dragging set the grid option:
+
+```python
+dashGridOptions = {"rowDragManaged": True}
+```
+
+### Dragging Animation
+
+To enable animation of the rows while dragging, set the grid option:
+
+```python
+dashGridOptions = {'animateRows': True}
+```
+
+### Suppress Move When Dragging
+
+By default, the managed row dragging moves the rows while you are dragging them. This effect might not be desirable due
+to your application design. To prevent this default behaviour, set the grid option:
+
+```python
+dashGridOptions = {"suppressMoveWhenRowDragging": True}
+```
+
+### Multi-Row Dragging
+
+It is possible to drag multiple rows at the same time, set the grid options:
+
+```python
+dashGridOptions = {"rowDragMultiRow": True, "rowSelection": "multiple"}
+```
 
 The example below shows simple managed dragging. The following can be noted:
 
-The first column has `rowDrag=True` which results in a draggable area being included in the cell.
-The property rowDragManaged is set, to tell the grid to move the row as the row is dragged.
-If a sort (click on the header) or filter (open up the column menu) is applied to the column, the draggable icon for row dragging is hidden. This is consistent with the constraints explained after the example.
+- The first column has `rowDrag=True` which results in a draggable area being included in the cell.
+- The property `rowDragManaged` is enabled, to tell the grid to move the row as the row is dragged.
+- If a sort (click on the header) or filter (open up the column menu) is applied to the column, the draggable icon for
+  row
+  dragging is hidden. This is consistent with the constraints explained after the example.
+
+When multi-row dragging is enabled:
+
+- When you select multiple items and drag one of them, all items in the selection will be dragged.
+- When you drag an item that is not selected while other items are selected, only the unselected item will be dragged.
+"""
+
+text2 = """
+The logic for managed dragging is simple and has the following constraints:
+
+- Works with [Client-Side](https://dashaggrid.pythonanywhere.com/clientside-data/overview) row model only; not with
```

Comment:
```suggestion
- Works with <dccLink href='/clientside-data/overview' children='Client-Side' /> row model only; not with
```
This type of link allows for navigating to a new page without refreshing the page.

Reply:
Yes indeed it is better to have a relative link, I tried 
`[Client-Side](/clientside-data/overview)` which works, but the `make_md()` function add `link_target="_blank"` to all links which open the link in a new tab...
Is it really a behavior we want?

And is there a difference between? :
`<dccLink href='/clientside-data/overview' children='Client-Side' />`
and
`<a href='/clientside-data/overview'>Client-Side</a>`

Reply:
Good question!  When you use the html `a`, it will navigate correctly but it will refresh the page.  The `dccLink` is actually the `dcc.Link` component so it doesn't refresh the page.   I did a forum post on this since it's a really cool feature that's only available with `dcc.Markdown`, and it's not in the dash-docs.
https://community.plotly.com/t/how-to-use-dcc-link-in-dcc-markdown-for-high-performance-links/66781